dot/digest/digest.go

```diff
@@ -110,7 +110,6 @@ func (h *Handler) toConsensusDigests(scaleVaryingTypes []scale.VaryingDataType)
 		}
 		digest, ok := digestValue.(types.ConsensusDigest)
 		if !ok {
-			h.logger.Errorf("digest type not supported: %T", digestValue)
```

Comment:
Actually... ideally... what you could do is a type `switch` with all digest type supported, and panic on the default case. That way we cannot forget to add a new digest type in the future. But really nitpicking I guess.